backend/services/recommendationService.js

```diff
@@ -1,14 +1,19 @@
 const skillsService = require('./skillsService');
 const placementsService = require('./placementService');
+const studentService = require('./studentService');
 
 module.exports = {
 
     //for employers
-    // getStudentRecommendationsForPlacement: async (placementID) => {   
-    //     let placementSkills = await placementService.getPlacementSkills(placementID);
-    //     let students = await studentService.getStudentsBySkills(placementSkills);
-    //     return students;
-    // },
+    getStudentRecommendationsForPlacement: async (placementID) => {   
+        let placementSkills = await placementsService.getPlacementSkills(placementID);
+        let studentIds = await studentService.getStudentsBySkills(placementSkills);
+        let students =  [];
+        for(let i = 0; i < studentIds.length; i++) {
+            students[i]= await studentService.getStudentProfile(studentIds[i].student_id);
```

Comment:
This is very bad!! It works in JavaScript but it's a wrong way to add to an array! Please use: 
`let student = await studentService[...];
students.push(student)`

Reply:
I fixed this here and elswhere I used it